### PR TITLE
Salief/add governor provider

### PR DESCRIPTION
**Pull request by salieflewis**

### CODE REVIEW

File: apps/next/pages/providers.tsx [hunk 1]
```diff
@@ -1,17 +1,19 @@
 import { Seo } from '@/components/Seo'
-import { ManagerProvider } from '@public-assembly/dao-utils'
+import { ManagerProvider, GovernorProvider } from '@public-assembly/dao-utils'
 import { isServerSide } from 'utils/helpers'
 import { PrintManagerProviderData } from '../components'
 
 function Providers() {
-  const tokenAddress = '0x8983ec4b57dbebe8944af8d4f9d3adbafea5b9f1' as `0x${string}`
+  const tokenAddress = '0xdf9b7d26c8fc806b1ae6273684556761ff02d422' as `0x${string}`
```
Comment (E-Rick) [inline apps/next/pages/providers.tsx:7]: Any string constants are best declared as all caps and snake case preferably in their own file for constants to keep track of them or at least contextually in the same file at the top.

File: packages/dao-utils/src/abi/index.ts [hunk 2]
```diff
@@ -1,2 +1,3 @@
 export * from './auctionAbi'
 export * from './managerAbi'
+export * from './governorAbi'
```
Comment (E-Rick) [inline packages/dao-utils/src/abi/index.ts:3]: why do we need these exports? we just export from the base index.ts

**Review by E-Rick — CHANGES_REQUESTED** [review]
Small changes but looks great. Thanks @salieflewis

File: packages/dao-utils/src/abi/index.ts (hunk 2 above)
Comment (salieflewis) [reply packages/dao-utils/src/abi/index.ts:3]: Stills allows you to access the abis with a simpler syntax. I've changed the import statements in the providers to reflect this.

File: packages/dao-utils/src/context/GovernorProvider.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
+import React, { useContext } from 'react'
+import { useContractRead, useContract } from 'wagmi'
+import { governorAbi } from '../abi/governorAbi'
+import { useManagerContext } from './ManagerProvider'
+
+type Hash = `0x${string}`
+
+export interface GovernorProviderProps {
+  children?: React.ReactNode
+  proposalId: `0x${string}`
```
[the remark this answers is not in the document]
Comment (salieflewis) [reply packages/dao-utils/src/context/GovernorProvider.tsx]: Done.

**Review by E-Rick — APPROVED** [review]
Thanks for making the changes

**Pull request merged**